syntax = "proto2";

message SetServoPosition {
  required int32 servo_front_right = 1;
  required int32 servo_front_left = 2;
  required int32 servo_back_right = 3;
  required int32 servo_back_left = 4;
  required int32 servo_yaw = 5;
}

message SetLedBrightness {
  required uint32 red = 1;
  required uint32 green = 2;
  required uint32 blue = 3;
}

